lib/testmap.py

```diff
@@ -128,15 +128,16 @@
     },
     'osbuild/cockpit-composer': {
         'main': [
-            'fedora-35',
             'fedora-36',
-            'fedora-36/firefox',
-            'rhel-8-7',
-            'rhel-9-1',
+            'fedora-37',
```

Comment:
Neither of these new tests ran in https://github.com/osbuild/cockpit-composer/pull/1568
Wouldn;t it be better to add them to manual, trigger them in the given PR and then enable them in main?

Reply:
Do you have any idea why the rhel tests aren't running but fedora-37 is? I just trigger them using my local checkout of bots with the updated testmap. That seems to work fine. Or is that why the tests are stuck pending?